README.md

```diff
@@ -1,18 +1,29 @@
 # .NET Core Samples
 
 This folder contains all the sample code that is part of any topic under
-the .NET Core documentation. There are several different projects that
+the .NET documentation. There are several different projects that
 are organized in sub-folders. These sub-folders are organized similar
-to the organization of the docs for .NET Core.
+to the organization of the docs for .NET.
 
 Some of the articles will have more than one sample associated with them. 
 
 The readme.md file for each sample will refer to the article so that
 you can read more about the concepts covered in each sample.
 
+There are two classes of samples in this repository:
+
+- **Samples** represent programs that demonstrate apoplication or library scenarios. These samples are typically larger, and often use more than one technology, feature, or toolkit.
+- **Snippets** represent small focused examples that demonstrate one feature or syntax. These should be no more than a single screen of code.
+
+In both cases, we verify the samples in our CI system.
+
+## Snippets
+
+Snippets are extracted from small programs that include the snippet. Snippets are all located in the top level **/snippets** folder. Snippets should follow all other sample guidance.
+
 ## Building a sample
 
-You build the samples using the .NET Core CLI. You can download the CLI from
+You build any .NET core sample using the .NET Core CLI. You can download the CLI from
 [the .NET Core home page](http://microsoft.com/net/core). Then, execute
```

Comment:
this url now is a get started page. not the best link now I'd say.

Reply:
updated to https://www.microsoft.com/net/download